src/sphinx_llm/tests/test_txt.py

```
@@ -412,6 +412,106 @@ def test_dirhtml_links_match_published_locations(tmp_path: Path):
+
+    app = Sphinx(
+        srcdir=str(source_dir),
+        confdir=str(source_dir),
+        outdir=str(tmp_path / "output"),
+        doctreedir=str(tmp_path / "doctrees"),
+        buildername="html",
+        warningiserror=False,
+        freshenv=True,
+    )
+
+    with pytest.raises(ExtensionError, match="llms_txt_source 'missing.rst'"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)